src/v1/models/AppState.js

```diff
@@ -29,7 +28,12 @@ const NEW_USER = '/img/security/unknown-device.png';
 const NEW_USER_IMAGE_DESCRIPTION = '';
 const UNDEFINED_USER_IMAGE_DESCRIPTION = '';
 const UNKNOWN_IMAGE_DESCRIPTION = '';
-const securityImageUrlTpl = hbs('{{baseUrl}}/login/getimage?username={{username}}');
```

Comment:
I believe we need to use the `hbs` templating function to prevent malicious usage

Reply:
Ah, okay got it. Handlebars was probably not the right thing to use to template this string since it's meant for HTML, so your solution of using `encodeURIComponent` makes sense